pkg/operation/botanist/component/etcd/etcd.go

```diff
@@ -434,11 +434,11 @@ func (e *etcd) Deploy(ctx context.Context) error {
 				},
 				LimitsRequestsGapScaleParams: hvpav1alpha1.ScaleParams{
 					CPU: hvpav1alpha1.ChangeParams{
-						Value:      pointer.StringPtr("1"),
+						Value:      pointer.StringPtr("2"),
 						Percentage: pointer.Int32Ptr(40),
 					},
 					Memory: hvpav1alpha1.ChangeParams{
-						Value:      pointer.StringPtr("1G"),
+						Value:      pointer.StringPtr("5G"),
```

Comment:
Why is it desired to increase by a ration of **5** (`1G` -> `5G`) while we increased the limits below only by **2** (`3G`- > `6G`)?

Reply:
This change was triggered because of https://github.tools.sap/kubernetes-live/issues-live/issues/250. Because of this I checked some of the largest of etcds in canary and live. I found the memory spikes of `~5G` the largest among them. Hence, the gap configuration absolute value of `5G`. And because the default requests are `1G`, increased the limits to `6G` (1+5). The original limits of `3G` seems to be arbitrary/historical from before the HVPA support for `LimitsRequestsGapScaleParams`.

The `LimitsRequestsGapScaleParams` were introduced to improve the VPA's [proportional scaling of limits](https://github.com/kubernetes/autoscaler/tree/master/vertical-pod-autoscaler#limits-control) because proportional scaling doesn't work well at both min and max ends of the scale because the absolute gap between requests and limits shrinks considerably at min and max ends of the scale. The `LimitsRequestsGapScaleParams` allows for both an absolute value as well as a percentage value to be configured for the gap between requests and limits. Whatever happens to be larger is taken. Typically, the absolute value applies at the lower end of the scale and the percentage applies at the higher end. In this case, the gap is guaranteed to be at least `5G`. It could be larger at higher ends of the scale if percentage quantity exceeds `5G`.

Reply:
Thanks for the explanation @amshuman-kr.